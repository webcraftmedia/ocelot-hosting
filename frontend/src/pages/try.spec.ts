
import { renderSuspended } from '@nuxt/test-utils/runtime'
import Page from './try.vue'
import { describe, expect, it } from 'vitest'

describe('Page: Try', () => {
  it('renders', async () => {
    const html = await (await renderSuspended(Page, { route: '/' })).html()
    expect(html).toMatchSnapshot()
  })
})