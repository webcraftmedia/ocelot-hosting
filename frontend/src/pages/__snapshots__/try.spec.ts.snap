// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Page: Try > renders 1`] = `
{
  "baseElement": <body>
    <div
      data-v-app=""
      id="__nuxt"
    >
      <!---->
    </div>
    <div>
      <div
        id="test-wrapper"
      >
        
        <!-- eslint-disable-next-line @intlify/vue-i18n/no-raw-text -->
        <section>
           Text: ... Ocelot credentials CTA -&gt; Try now -&gt; Redirect 
        </section>
        
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      id="test-wrapper"
    >
      
      <!-- eslint-disable-next-line @intlify/vue-i18n/no-raw-text -->
      <section>
         Text: ... Ocelot credentials CTA -&gt; Try now -&gt; Redirect 
      </section>
      
    </div>
  </div>,
  "debug": [Function],
  "emitted": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "html": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "setupState": undefined,
  "unmount": [Function],
}
`;
